parse_hut.py

```diff
@@ -35,16 +36,22 @@ def parse_usages(usage_list):
         if not line:
             continue
         if line.startswith('('):
-            idx, page_name = line.split('\t')
-            idx = int(idx.lstrip('(').rstrip(')'), 16)
+            r = _parse('({idx:x})\t{page_name}', line)
+            assert(r is not None)
+            idx = r['idx']
+            page_name = r['page_name']
             continue
-        usage, name = line.split('\t')
-        if 'reserved' in name.lower():
-            continue
-        if '-' in usage:
+        if _parse('{:x}-{:x}\t{}', line):
             print(line)
             continue
-        usages[int(usage, 16)] = name
+
+        r = _parse('{usage:x}\t{name}', line)
+        assert r is not None
+        if 'reserved' in r['name'].lower():
```

Comment:
This should be moved up as we don't really want to show the reserved ranges in the `if _parse('{:x}-{:x}\t{}', line):` statement above